model_analyzer/config/generate/perf_analyzer_config_generator.py

```diff
@@ -72,10 +74,10 @@ def __init__(
 
         # All configs are pregenerated in _configs[][]
         # Indexed as follows:
-        #    _configs[_curr_batch_size_index][_curr_parameter_index]
+        #    _configs[_curr_non_parameter_index][_curr_parameter_index]
         #
+        self._curr_non_parameter_index = 0
```

Comment:
Sorry, kinda struggling to follow some of this.

Why did batch size go away and why is it called non-parameter. What does (non-)parameter mean I this context?